pkg/azure/credential/ucpcredentials.go

```diff
@@ -49,6 +49,9 @@ type UCPCredentialOptions struct {
 
 	// ClientOptions is the options for azure client.
 	ClientOptions *azcore.ClientOptions
+
+	// TokenFilePath is the path to the azure token file (for use with Azure workload identity)
+	TokenFilePath string
```

Comment:
added this to make this file unit testable. the azure sdk library will use the environment variable in the real world